devtools/startup/DevToolsStartup.jsm

```diff
@@ -1856,7 +1816,7 @@ async function reloadAndStopRecordingTab(gBrowser) {
     return;
   }
 
-  const { recordingId } = data;
+  const recordingId = data.id;
```

Comment:
This was the main fix we missed in our testing, because Dan's local build was crashing for some reason.